insights/specs/datasources/eap_reports.py

```diff
@@ -0,0 +1,46 @@
+"""
+Custom datasource to get all updated/created eap reports in last 24 hours
+"""
+import os
+import time
+import logging
+
+from insights.core.context import HostContext
+from insights.core.plugins import datasource
+from insights.core.exceptions import SkipComponent
+from insights.core.spec_factory import TextFileProvider
+
+log = logging.getLogger(__name__)
+
+
+def get_recent_files(target_path, last_modify_hours):
```

Comment:
How about moving this function to a single module under the `datasources` folder, and e.g. name it as `recent_files.recent_files_of` (recent_files_of() in the recent_files module)?